packages/@aws-cdk/aws-codeguruprofiler/test/integ.profiler-group-import-functions.ts

```diff
@@ -30,6 +30,12 @@ const importedGroupWithImplicitlySetName = ProfilingGroup.fromProfilingGroupName
   profilingGroup2.profilingGroupName,
 );
 
+const importedGroupFromArn = ProfilingGroup.fromProfilingGroupArn(
+  stack,
+  'ImportedProfilingGroupFromArn',
+  'arn:aws:codeguru-profiler:us-east-1:1234567890:profilingGroup/MyAwesomeProfilingGroup',
```

Comment:
How did this test pass with the dummy account number? Was the test actually run or was it dry-run?

Reply:
Also, where is this profiling group created? It doesn't look like it exists.

Reply:
> Was the test actually run or was it dry-run?

Yes, this was executed

> How did this test pass with the dummy account number? 
> Also, where is this profiling group created? It doesn't look like it exists.

all the `fromXArn` functions do is parse the resource identifiers from the arn and injects them directly into the template, so it doesn't really matter if the resource actually exists. If you look at `ProfilingGroupTestStack.template.json` you can see that its directly inlining the values.

It's probably unnecessary to even have an integ test for this given that to test it you don't actually need to create a resource. Most of these functions don't have integ tests, though there's [at least one other example](https://github.com/aws/aws-cdk/blob/129c695ae7d31613b4b8a48affff43c9e8a734ae/packages/%40aws-cdk/aws-codebuild/test/integ.docker-registry.lit.ts#L10)

Reply:
If it's not actually validating the account number or region, can you use a dummy value for the region so that we can verify that this isn't coming from running the tests in us-east-1? That's the default region for test stacks so showing that it's not that would be a better test.

Reply:
ack, have updated the test to not use `us-east-1`